Directory.Build.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <!-- $(RepoRoot) is normally set globally and Arcade overrides it to ensure a trailing slash. -->
-    <RepoRoot Condition=" '$(RepoRoot)' == '' ">$([MSBuild]::EnsureTrailingSlash('$(MSBuildThisFileDirectory)'))</RepoRoot>
+    <RepoRoot Condition=" '$(RepoRoot)' == '' OR !HasTrailingSlash('$(RepoRoot)') ">$(MSBuildThisFileDirectory)</RepoRoot>
```

Comment:
Note: I was mistaken thinking `EnsureTrailingSlash(...)` was needed

Reply:
@rainersigwald one question: Why is this able to override `%RepoRoot%` in @SteveSandersonMS's #32615 scenario (under "Older report")❔ Are environment variables able to be overridden in `msbuild` files even though they're otherwise treated as global properties❔

Reply:
Environment variables are treated as normal properties, not global properties, so they're mutable, they just have an initial value at the very beginning of evaluation.